src/spec.c

```diff
@@ -1743,6 +1755,9 @@ int Indexes_RdbLoad(RedisModuleIO *rdb, int encver, int when) {
     }
   }
   return REDISMODULE_OK;
+
+cleanup:
+  return REDISMODULE_ERR;
```

Comment:
Review by Meir: need to cleanup all `IndexSpec` created in this call when a short-read occurs,
e.g., if 2 `IndexSpec` were read successfully and the third one fails due to short-read, need to cleanup all three.

Reply:
@oshadmi Is it possible to give an ETA (roughly)?

Reply:
@tw-bert I apologize for my late response. ETA hopefully this week (or maybe next week) once code is reviewed and probably amended accordingly.

Reply:
@oshadmi Thanks for the update!